tests/ch.elexis.core.findings.util.test/src/ch/elexis/core/findings/util/fhir/transformer/AllTransformerTests.java

```diff
@@ -64,4 +69,14 @@ public static IModelService getCoreModelService() {
 		return coreModelService;
 	}
 
+	public static IElexisEntityManager getElexisEntityManager() {
+		return OsgiServiceUtil.getService(IElexisEntityManager.class).get();
+	}
+	public static IAppointmentService getAppointmentService() {
+	    return OsgiServiceUtil.getService(IAppointmentService.class).get();
+	}
+
+	public static IConfigService getConfigService() {
+	    return OsgiServiceUtil.getService(IConfigService.class).get();
+	}
```

Comment:
Es wäre gut diese Services zumindest in einer statischen Variable zu halten, und nur wenn diese null ist eine neue ServiceReferenz zu erzeugen.